pkg/users/usersGet_test.go

```diff
@@ -65,56 +71,40 @@ func mockUsersServlet(t *testing.T, writer http.ResponseWriter, request *http.Re
 				"last-login": "2024-10-28T15:32:49.546029Z",
 				"client-name": "rest-api"
 			}
-		]
+		],
+		"synthetic" : {
+			"role": {
+				"metadata" : {
+					"name" : "admin"
+				}
+			}
+		}
```

Comment:
We could create objects using the generated beans and marshal them into JSON strings instead of hard-coding JSON response structures in the tests? Just a thought!

Reply:
Yes. I largely refactored this test class as it was to move the test data into the test methods which used that data. Hated what was going on there before. But will consider this.